### DISCUSSION TITLE
In-game "player info" overlay doesn't show name changes

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#24731] [Q&A] [answered]
Lazer doesn't show this feature. On the website on a player's profile, if they have changed their name, their previous names are listed with an icon near their current name.

### Screenshots

Website
![web](https://github.com/ppy/osu/assets/101510334/427c27d9-35d8-485c-9f10-5ceb096e422d)

Website; hovering over the icon expands it to show the player's username history
![web-expanded](https://github.com/ppy/osu/assets/101510334/002d1ed9-fe93-401b-bc59-131bd7b8b561)

-------------
Lazer; there is no icon
![lazer](https://github.com/ppy/osu/assets/101510334/08a2d663-0c1a-4153-ae6d-b4e9c349549d)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/24737
